src/NServiceBus.Core/Unicast/Config/UnicastBus.cs

```diff
@@ -20,7 +20,7 @@ internal UnicastBus()
             Defaults(s =>
             {
                 var section = s.GetConfigSection<UnicastBusConfig>();
-                if (section.TimeoutManagerAddress != null)
+                if (section?.TimeoutManagerAddress != null)
```

Comment:
Nitpicking: how about
```
var address = section?.TimeoutManagerAddress
if (address != null) {
   s.Set("TimeoutManagerAddress", address);
}
```
one access to section less (not for performance reasons, more about styling)